app/controllers/api/v1/pets_controller.rb

```diff
@@ -1,4 +1,4 @@
-class Api::V1::PetsController < ApplicationController
+class Api < V1 < PetsController < ApplicationController
```

Comment:
This is isn't right. `PetsController` is supposed to be a class of the modules `Api::V1`. In fact, this might break the app. Keep it the way it was before. 😎